openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -446,6 +449,10 @@ The request parameter `authorization_details` defined in Section 2 of [@!RFC9396
 * `credential_configuration_id`: REQUIRED when `format` parameter is not present. String specifying a unique identifier of the Credential being described in the `credential_configurations_supported` map in the Credential Issuer Metadata as defined in (#credential-issuer-parameters). The referenced object in the `credential_configurations_supported` map conveys the details, such as the format, for issuance of the requested Credential. This specification defines Credential Format specific Issuer Metadata in (#format-profiles). It MUST NOT be present if `format` parameter is present.
 * `format`: REQUIRED when `credential_configuration_id` parameter is not present. String identifying the format of the Credential the Wallet needs. This Credential Format Identifier determines further claims in the authorization details object needed to identify the Credential type in the requested format. This specification defines Credential Format Profiles in (#format-profiles). It MUST NOT be present if `credential_configuration_id` parameter is present.
 
+Additional `authorization_details` data fields MAY be defined and used,
+as described in [@!RFC9396].
+The Wallet MUST ignore any unrecognized parameters.
```

Comment:
Since the `authorization_details` in the authorization request is created by the wallet itself, does it make sense to state that "The Wallet MUST ignore any unrecognized parameters"? Perhaps this sentence should be in the `token-response` section, where the `authorization_details` is generated by the AS.

Reply:
@pmhsfelix Thanks.  I corrected the party that must ignore `authorization_details` extensions.

Reply:
Should we also add the equivalent sentence on the `authorization_details` returned in the token response, where now is the wallet that MUST ignore unrecognized parameters put there by the AS?